### PR TITLE
Stop assuming gender.

### LINKED ISSUE #8281: Medical Splints
**Problem Description**:
What is the problem?
When you medical splint someone the gender for the splint seems to stay as the masculine version. For example:
She has a splint on his left leg!
She has a splint on his right leg!

it is a she and the splint is on his leg.

**What did you expect to happen**: 
Why do you think this is a bug?
to be the correct gender verb.

**What happened instead**:
How is what happened different from what you expected?
Its stuck as male. i splinted their gender.

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
Not that bad just a minor bug.

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
Just splint a female character and see.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.
nope they had a broken leg and none broken i splinted both to check.

### PR DESCRIPTION
**Pull request by Anasari**
Fix #8281

Stop some cases of assuming gender in the code.

🆑 
fix: Some genders are no longer assumed.
🆑

### CODE REVIEW

**Comment by KasparoVy** [comment]
Helecopterkin.

**Comment by TDSSS** [comment]
Can you change this in the zeroth law added by emagging borgs, too? It is currently something like 'Only [Name] and people **he** designates are Syndicate agents', even if you're female.

File: code/modules/mob/living/carbon/superheroes.dm [hunk 1]
```diff
@@ -188,7 +188,7 @@
 			switch(progress)
 				if(1)
 					to_chat(user, "<span class='notice'>You begin by introducing yourself and explaining what you're about.</span>")
-					user.visible_message("<span class='danger'>[user]'s introduces himself and explains his plans.</span>")
+					user.visible_message("<span class='danger'>[user]'s introduces himself and explains \his plans.</span>")
```
Comment (MarcellusPye) [inline code/modules/mob/living/carbon/superheroes.dm]: [user]'s introduces himself and explains her plans.

File: code/game/objects/items/weapons/cosmetics.dm [hunk 2]
```diff
@@ -143,7 +143,7 @@
 				user.visible_message("<span class='warning'>[user] starts to shave their head with \the [src].</span>", \
 				"<span class='warning'>You start to shave your head with \the [src].</span>")
 				if(do_after(user, 50 * toolspeed, target = H))
-					user.visible_message("<span class='notice'>[user] shaves his head with the [src].</span>", \
+					user.visible_message("<span class='notice'>[user] shaves \his head with the [src].</span>", \
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/cosmetics.dm]: This `the` and the above `the` are missing backslashes

**Comment by HugoLuman** [comment]
The funny thing is, I could totally see borgs and AIs trying to get out of certain laws due to the wrong pronouns being used.

**Comment by HugoLuman** [comment]
The funny thing is, I could totally see borgs and AIs trying to get out of certain laws due to the wrong pronouns being used.

File: code/game/objects/items/weapons/cosmetics.dm [hunk 3]
```diff
@@ -143,8 +143,8 @@
 				user.visible_message("<span class='warning'>[user] starts to shave their head with \the [src].</span>", \
 				"<span class='warning'>You start to shave your head with \the [src].</span>")
 				if(do_after(user, 50 * toolspeed, target = H))
-					user.visible_message("<span class='notice'>[user] shaves his head with the [src].</span>", \
-					"<span class='notice'>You finish shaving with the [src].</span>")
+					user.visible_message("<span class='notice'>[user] shaves \his head with \the [src].</span>", \
+					"<span class='notice'>You finish shaving with\the [src].</span>")
```
Comment (Crazylemon64) [inline code/game/objects/items/weapons/cosmetics.dm]: You're missing a space here

File: code/modules/mob/living/carbon/superheroes.dm [hunk 4]
```diff
@@ -214,7 +214,7 @@
 		recruiting = 0
 		to_chat(user, "<span class='notice'>You have recruited <b>[target]</b> as your henchman!</span>")
 		to_chat(target, "<span class='deadsay'><b>You have decided to enroll as a henchman for [user]. You are now part of the feared 'Greyshirts'.</b></span>")
-		to_chat(target, "<span class='deadsay'><b>You must follow the orders of [user], and help him succeed in his dastardly schemes.</span>")
+		to_chat(target, "<span class='deadsay'><b>You must follow the orders of [user], and help him succeed in \his dastardly schemes.</span>")
```
[the remark this answers is not in the document]
Comment (HugoLuman) [reply code/modules/mob/living/carbon/superheroes.dm]: Holy crap, somehow this got saved as a draft or something and not posted when I meant to

**Pull request merged**